src/_locales/en_US/messages.json

```diff
@@ -135,6 +147,15 @@
   "viewSettings-termsAndConditionsLink": {
     "message": "Terms & Conditions"
   },
+  "viewBetaDecision": {
+    "message": "TBD"
+  },
+  "viewBetaDecisionButton": {
+    "message": "Learn more"
+  },
+  "viewBetaDecisionAlreadyMade": {
+    "message": "TBD decision made?"
```

Comment:
Perhaps put some real placeholder text in these in case we want to merge?

Reply:
We don't have this yet.